### PR TITLE
Adding fuel changes

### PR DESCRIPTION
**Pull request by frangrech**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -54,9 +54,3 @@ You can hire us by submiting an issue and fund the issue on [IssueHunter](https:
 
 Developers: @liangping @dingyiming
 
-## Changes from Original Ping.Pub
-
-This version of Ping.Pub was forked from from commit https://github.com/ping-pub/explorer/commit/603d7ce2, with the following changes applied:
-
-- Removed Sponsors from DefaultLayout.vue
-
```
Comment (dillu24) [inline README.md]: We need to keep this. Can you please keep it and populate it with your list of changes?

File: README.md [hunk 2]
```diff
@@ -58,5 +58,10 @@ Developers: @liangping @dingyiming
 
 This version of Ping.Pub was forked from from commit https://github.com/ping-pub/explorer/commit/603d7ce2, with the following changes applied:
 
-- Removed Sponsors from DefaultLayout.vue
```
Comment (dillu24) [inline README.md:61]: Can you please keep this?

File: chains/mainnet/fuel.json
Comment (dillu24) [inline chains/mainnet/fuel.json:1]: Do we need to also add one for testnet?

**Review by dillu24 — CHANGES_REQUESTED** [review]
Reviewed

File: chains/mainnet/fuel.json
Comment (dillu24) [reply chains/mainnet/fuel.json:1]: I guess no right coz we don't want two entries on the explorer,  we just want to modify the chain json.

Comment (frangrech) [reply chains/mainnet/fuel.json:1]: We can add the spec for testnet and use the same explorer URL, then the user can choose between Fuel Mainnet and Fuel Testnet, up to you

Comment (dillu24) [reply chains/mainnet/fuel.json:1]: Lets keep separate for now and I'll bring it up with @kribby

**Pull request merged**